util/customTransformer-v1.js

```diff
@@ -43,16 +43,18 @@ function calculateMsFromIvmTime(value) {
 async function userTransformHandlerV1(
   events,
   userTransformation,
-  libraryVersionIds
+  libraryVersionIds,
+  testMode = false
 ) {
   /*
  Transform VM aquire mode is
  on demand if env variable ON_DEMAND_ISOLATE_VM = true | True | TRUE,
  Pooled otherwise
 */
-  const isAcquireTransformerIsolatedVMMode = process.env.ON_DEMAND_ISOLATE_VM
-    ? process.env.ON_DEMAND_ISOLATE_VM.toLowerCase() === "true"
-    : false;
+  const isAcquireTransformerIsolatedVMMode =
+    process.env.ON_DEMAND_ISOLATE_VM && !testMode
```

Comment:
Fix the condition. Test mode should always avoid pool.

Reply:
done